core/server/worker/src/main/java/alluxio/worker/page/PagedBlockStore.java

```diff
@@ -507,4 +510,33 @@ private int getDirIndexOfBlock(long blockId) {
         .getDir()
         .getDirIndex();
   }
+
+  private void removeBlockInternal(long sessionId, long blockId, long timeoutMs)
+      throws IOException {
+    Optional<BlockLock> optionalLock =
+        mLockManager.tryAcquireBlockLock(sessionId, blockId, BlockLockType.WRITE,
+        timeoutMs, TimeUnit.MILLISECONDS);
+    if (!optionalLock.isPresent()) {
+      throw new DeadlineExceededException("Cannot acquire lock to remove block");
```

Comment:
Include the block ID in the exception message.

Reply:
Done.